### PR TITLE
adds 500.md template and error handling in production

### PR DESCRIPTION
**Pull request by nunofaria11**

### CODE REVIEW

**Comment by butlerx** [comment]
just need you to handle the conflicts

**Comment by nunofaria11** [comment]
Will get to this later today or tomorrow.

File: smtpTransport.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 const nodemailer = require('nodemailer');
 
-const fs = require('fs');
-const yaml = require('js-yaml');
-
-const configFile = fs.readFileSync('./_config.yml', 'utf8');
-const config = yaml.safeLoad(configFile).server;
+const config = require('./config.json');
```
Comment (butlerx) [inline smtpTransport.js]: this doesnt exist any more

**Comment by nunofaria11** [comment]
I'm now having problems with a missing `.env` file (though a `.env.example` exists) - should I rename it?

**Comment by nunofaria11** [comment]
Already saw the README

**Pull request merged**